admin.php

```diff
@@ -7,39 +7,124 @@
+
+    /**
+     * Template folder directory.
+     */
+    const TEMPLATE_DIR = __DIR__ . '/tpl';
+
+    const TEMPLATE_FILE = 'admin.tpl';
+
+    public function getMenuSort()
+    {
+        return self::MENU_SORT;
     }
 
     /**
-     * @return bool true if only access for superuser, false is for superusers and moderators
+     * Admins or higher can access this plugin.
+     *
+     * You do not need to be a superuser to access this plugin.
      */
-    public function forAdminOnly() {
+    public function forAdminOnly()
+    {
         return true;
     }
 
     /**
-     * Should carry out any processing required by the plugin.
+     * HTML output (gets generated by twig).
      */
-    public function handle() {
+    public function html()
+    {
+        global $conf;
+        $pagesWithoutTitle = PageGetter::getPagesWithoutTitle();
+        $variables = $this->getVariablesForTemplate($pagesWithoutTitle);
+
+        $loader = new Twig_Loader_Filesystem(self::TEMPLATE_DIR);
+        $twig = new Twig_Environment($loader);
+
+        $initLanguage = $this->getLang('');
+
+        $twig->addGlobal('languageText', $this->lang);
+
+        echo $twig->render(self::TEMPLATE_FILE, $variables);
+    }
+
+
+    /**
+     * @param $pagesWithoutTitle array of pages without title.
+     * @return array variables for twig template.
+     */
+    private function getVariablesForTemplate($pagesWithoutTitle)
+    {
+
+        // Generate variables based on page data
+        foreach ($pagesWithoutTitle as $key => $page) {
+            $metadata[] = p_get_metadata($page);
+            $url[] = wl($page, '', true);
+
+            // Make new DateTime Object and Format it
+            $modifiedTimeStamp[] = $this->formatTime($metadata[$key]['last_change']['date']);
+        }
+
+        // Put variables to array
+        $variables = array(
+            'pagesWithoutTitle' => $pagesWithoutTitle,
+            'metadata' => $metadata,
+            'urls' => $url,
+            'timestamp' => $modifiedTimeStamp,
+            'imageUrl' => self::EDIT_IMAGE_URL,
+        );
+
+        return $variables;
+    }
+
+    /**
+     * @param $unixTime int from now.
+     * @return string Time as Formatted string with timezone.
+     */
+
```

Comment:
Docblock should stick to method declaration.

Reply:
Thanks for noticing!